src/frontend/package.json

```diff
@@ -6,15 +6,20 @@
   "scripts": {
     "serve": "npm run build && serve -s build -l tcp://0.0.0.0:3000",
     "start": "node scripts/start.js",
-    "build": "node scripts/build.js",
+    "build": "node --max_old_space_size=4096 scripts/build.js",
```

Comment:
do we still need this flag?

Reply:
yes! We still need to allocate more memory to `npm` to build the app on top of increasing the FE container size!